src/goose/cli/main.py

```diff
@@ -208,7 +208,10 @@ def session_resume(name: Optional[str], profile: str, log_level: str) -> None:
 @click.option("--profile")
 @click.option("--log-level", type=LOG_CHOICE, default="INFO")
 @click.option("--resume-session", is_flag=True, help="Resume the last session if available")
-def run(message_file: Optional[str], profile: str, log_level: str, resume_session: bool = False) -> None:
+@click.option("--tracing", is_flag=True, required=False)
+def run(
+    message_file: Optional[str], profile: str, log_level: str, resume_session: bool = False, tracing: bool = False
```

Comment:
:nit: this method signature is a little long if we add a trailing comma `ruff` will forcibly newline all of these (e.g. `tracing: bool = False,`)